wreck/wreck.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for wreck 7.0.0
+// Project: https://github.com/hapijs/wreck
+// Definitions by: Marcin Porębski <http://github.com/marcinporebski>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+
+declare module "wreck"
+{
+    import http = require('http');
+    import stream = require('stream');
+
+
+    interface IWreckObject
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.